production.Dockerfile

```diff
@@ -25,6 +25,15 @@ FROM node:18.12.1-bullseye-slim AS frontend-build
 WORKDIR /code
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
+COPY transpiler/ transpiler/
+WORKDIR /code/transpiler
+RUN ls && corepack enable && pnpm --version && \
+    mkdir /tmp/pnpm-store && \
+    pnpm install --frozen-lockfile --store-dir /tmp/pnpm-store && \
+    pnpm build && \
+    rm -rf /tmp/pnpm-store
+
+WORKDIR /code
 COPY package.json pnpm-lock.yaml ./
 RUN corepack enable && pnpm --version && \
```

Comment:
Now we `corepack enable` etc. twice in one stage, which seems wasteful. I think it'd be pretty nice if `transpiler-build` was a separate stage.